src/time_advance_tests.cpp

```diff
@@ -136,7 +136,7 @@ TEMPLATE_TEST_CASE("time advance - diffusion 2", "[time_advance]", double,
     auto const full_grid = false;
     parser const parse(
         pde_choice, fk::vector<int>(std::vector<int>(num_dims, level)), degree,
-        cfl, full_grid, parser::DEFAULT_MAX_LEVEL, num_steps);
+        cfl, full_grid, parser::DEFAULT_MAX_LEVEL, -1, num_steps);
```

Comment:
```suggestion
        cfl, full_grid, parser::DEFAULT_MAX_LEVEL, parser::DEFAULT_MIXED_GRID_GROUP, num_steps);
```
repeat change for other instances of `-1` in this file.